### DISCUSSION TITLE
Github Copilot not working in PyCharm: Asking to re-enter API Token

### DISCUSSION THREAD

**Discussion by AAdewunmi** [community/community#7851] [Copilot Conversations] [answered]
Hi,
I installed Github Copilot in PyCharm 2021.2.3, a week ago. 
Installation was successful after entering Github Copilot API Token.
Wanted to use PyCharm this morning (18 November, 2021) and found 
Copilot disabled and a pop-up request to enter API Token. 
Tried to re-enter previous key but it did not work. Any fix suggestions?

<img width="440" alt="Screenshot 2021-11-18 at 10 19 25" src="https://user-images.githubusercontent.com/15172744/142400719-a85d81fd-b92f-4f17-842b-2f623dcbc6d7.png">

**Comment by AAdewunmi** [comment, marked as answer]
I have reconnected my Copilot service in PyCharm using @jansorg's suggestion 😎
See below:
(https://github.com/github/feedback/discussions/7723#discussioncomment-1644137)

**Comment by jansorg** [comment]
@AAdewunmi Actually, macOS is prompting on behalf of the plugin to access your keychain, where the API token is stored. We're considering changes to avoid this rather intrusive dialog.

**Comment by AAdewunmi** [comment]
@jansorg, thanks for the update.
